index.html

```diff
@@ -200,5 +200,37 @@ <h1>About</h1>
                         </div>
                     </div>
                 </section>
+
+                <section class="contactForm">
+                    <div class="contactLogo"><img src="./images/logo.png"
+                            alt="Logo"></div>
+                    <div class="content">
+                        <h1>Get started</h1>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+                            elit, sed do eiusmod tempor incididunt ut labore et
+                            dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/mjvzkrzj"  method="POST">
+                        <input type="text" required id="fullName" placeholder="Your name"
+                            name="Full Name" maxlength="30">
+                        <input type="email" required="required" name="" Email id="myEmail"
+                            placeholder="Your email">
+                        <textarea name="subject" id="msg" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <input type="submit" value="Start collaboration" id="button">
```

Comment:
- Your button has little variations from the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 design template. Please reference the image attached below.

Your project:
![goodjob button](https://user-images.githubusercontent.com/73607512/198970928-ea86febe-94b6-428f-bb36-97a3ec2b7000.PNG)

Design Template:
![figi](https://user-images.githubusercontent.com/73607512/198971012-0dda30c4-8530-4801-b118-2bc32a3fec0e.PNG)

Reply:
I did stick to the Figma design.
![Screenshot 2022-10-31 at 11 01 20](https://user-images.githubusercontent.com/37203742/198982886-b3403d5d-8f25-467f-9689-8925868b76cb.png)
my button is to display this background color when hovering: `background: #1a2236;` as directed by the _Figma design_

![Screenshot 2022-10-31 at 11 05 41](https://user-images.githubusercontent.com/37203742/198983583-d6aa691a-abc7-4419-8d11-8df1a6bd6f4c.png)
![Screenshot 2022-10-31 at 11 06 03](https://user-images.githubusercontent.com/37203742/198983590-354ecdaf-0516-44ba-9157-f2047456e25d.png)